lib/progress.c

```diff
@@ -229,16 +229,16 @@ void Curl_pgrsStartNow(struct Curl_easy *data)
  * need to wait until we're back under the speed limit, if needed.
  *
  * The way it works is by having a "starting point" (time & amount of data
- * transferred by then) used in the speed computation, to be used instead of the
+ * transfered by then) used in the speed computation, to be used instead of the
  * start of the transfer.
  * This starting point is regularly moved as transfer goes on, to keep getting
- * accurate values (instead of average over the entire transfer).
+ * accurate values (instead of average over the entire tranfer).
```

Comment:
Those were fine spelling fixes, it is just that the fix made the lines longer than allowed so it triggered checkscr errors. They need to wrap the line earlier too to become a perfect fix.